### PR TITLE
[CUBRIDQA-40] implement upgrade on each node for ctp

### PR DESCRIPTION
**Pull request by supler**
@fanzaiqiang 

implement ctp upgrade on each node, please review it. thanks

### CODE REVIEW

File: CTP/shell/src/com/navercorp/cubridqa/shell/deploy/DeployOneNode.java [hunk 1]
```diff
@@ -199,44 +197,17 @@ private void deploy_linux_by_cubrid_common() {
...
+		String ctpHome = context.getToolHome();
 		
 		ShellInput scripts = new ShellInput();
-		scripts.addCommand("echo 'BEGIN TO UPGRADE CQT'");
-		scripts.addCommand("cd $HOME/CQT");
-		scripts.addCommand("sh upgrade.sh 2>&1");
+		scripts.addCommand("echo 'BEGIN TO UPGRADE CTP'");
+		scripts.addCommand("export SKIP_UPGRADE=" + enableSkipUpgrade);
+		scripts.addCommand("export CTP_BRANCH_NAME=" + branchName);
+		scripts.addCommand("cd " + ctpHome);
```
Comment (fanzaiqiang) [inline CTP/shell/src/com/navercorp/cubridqa/shell/deploy/DeployOneNode.java]: The ctpHome in test node is different from controller node.

So in test node, just use $HOME/CTP.

File: CTP/shell/src/com/navercorp/cubridqa/shell/main/Context.java [hunk 2]
```diff
@@ -126,6 +130,8 @@ public void reload() throws IOException{
 		this.cubridPackageUrl = getProperty("main.testbuild.url");
 
 		this.serviceProtocolType = getProperty("main.service.protocol", "ssh").trim().toLowerCase();
+		this.enableSkipUpgrade = getPropertyFromEnv("SKIP_UPGRADE", "0");
```
Comment (fanzaiqiang) [inline CTP/shell/src/com/navercorp/cubridqa/shell/main/Context.java]: The default value of SKIP_UPGRADE is 1.

**Comment by supler** [comment]
@fanzaiqiang 

fix comments from you, thanks

File: CTP/shell/src/com/navercorp/cubridqa/shell/deploy/DeployOneNode.java [hunk 3]
```diff
@@ -200,13 +200,12 @@ private void deploy_build_on_linux() {
...
 		String enableSkipUpgrade = context.getEnableSkipUpgrade();
 		String branchName = context.getCtpBranchName();
-		String ctpHome = context.getToolHome();
 		
 		ShellInput scripts = new ShellInput();
 		scripts.addCommand("echo 'BEGIN TO UPGRADE CTP'");
 		scripts.addCommand("export SKIP_UPGRADE=" + enableSkipUpgrade);
 		scripts.addCommand("export CTP_BRANCH_NAME=" + branchName);
-		scripts.addCommand("cd " + ctpHome);
+		scripts.addCommand("cd $HOME/CTP");
```
Comment (fanzaiqiang) [inline CTP/shell/src/com/navercorp/cubridqa/shell/deploy/DeployOneNode.java]: It's better to ensure further:
chmod u+x upgrade.sh

**Comment by supler** [comment]
@fanzaiqiang 

change upgrade script as executable, thanks

**Comment by fanzaiqiang** [comment]
@supler 

Please go ahead to merge them. Thank you.

**Pull request merged**